### ISSUE TITLE
-Wimplicit-fallthrough in sound/soc/qcom/qdsp6/topology.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1495]
```
sound/soc/qcom/qdsp6/topology.c:465:3: warning: unannotated fall-through between switch labels [-Wimplicit-fallthrough]
                default:
                ^
sound/soc/qcom/qdsp6/topology.c:465:3: note: insert 'break;' to avoid fall-through
                default:
                ^
                break;
1 warning generated.
```

Patch submitted: https://lore.kernel.org/r/20211027190823.4057382-1-nathan@kernel.org/

**Comment by nathanchance** [comment]
Patch accepted: https://git.kernel.org/broonie/sound/c/c6c203bc4dfed6812cf77e7737074b9cff8dd78d

**Comment by nathanchance** [comment]
Fixed in next-20211029.
